### ISSUE TITLE
Updating python-minimal

### ISSUE THREAD

**Issue by ghost** [Jarli01/xenorchestra_installer#77]
**Describe the bug**
I am using Ubuntu 20 LTS. During the installation the terminal tolled me, that it is not possible to use python-minimal, but rather python2-minimal. Would it be possible to change it in your installation skript?

**To Reproduce**
1. Fresh install of 20 LTS
2. Open Terminal
3. Issue the command curl https://raw.githubusercontent.com/Jarli01/xenorchestra_installer/master/xo_install.sh | bash
4. During installation it will pop up in the terminal

**Desktop/Server details (please complete the following information):**
 - OS: Ubuntu 20 LTS

**Comment by Jarli01** [comment]
Python2-minimal is end of life, pyhton3-minimal should work, we actually have an open issue here about this but we haven't had a chance to fully vet python3-minimal.

**Comment by Jarli01** [comment]
Closing per the most recent merge PR

**Comment by Danp2** [comment]
FYI, I am receiving errors regarding python when attempting to build a new VM --
```
yarn install v1.22.4
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
warning url-loader@1.1.2: Invalid bin field for "url-loader".
info fsevents@1.2.13: The platform "linux" is incompatible with this module.
info "fsevents@1.2.13" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@2.1.3: The platform "linux" is incompatible with this module.
info "fsevents@2.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
[6/12] ⢀ argon2
[-/12] ⢀ waiting...
[-/12] ⢀ waiting...
[-/12] ⢀ waiting...
warning Error running install script for optional dependency: "/opt/xen-orchestra/node_modules/argon2: Command failed.
Exit code: 1
Command: node-pre-gyp install --fallback-to-build
Arguments:
Directory: /opt/xen-orchestra/node_modules/argon2
Output:
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.13.0
node-pre-gyp info using node@12.17.0 | linux | x64
node-pre-gyp WARN Using request for node-pre-gyp https download
node-pre-gyp info check checked for \"/opt/xen-orchestra/node_modules/argon2/lib/binding/argon2.node\" (not found)
node-pre-gyp http GET https://github.com/ranisalt/node-argon2/releases/download/v0.25.1/argon2-v0.25.1-node-v72-linux-x64-glibc.tar.gz
node-pre-gyp http 404 https://github.com/ranisalt/node-argon2/releases/download/v0.25.1/argon2-v0.25.1-node-v72-linux-x64-glibc.tar.gz
node-pre-gyp WARN Tried to download(404): https://github.com/ranisalt/node-argon2/releases/download/v0.25.1/argon2-v0.25.1-node-v72-linux-x64-glibc.tar.gz
node-pre-gyp WARN Pre-built binaries not found for argon2@0.25.1 and node@12.17.0 (node-v72 ABI, glibc) (falling back to source compile with node-gyp)
node-pre-gyp http 404 status code downloading tarball https://github.com/ranisalt/node-argon2/releases/download/v0.25.1/argon2-v0.25.1-node-v72-linux-x64-glibc.tar.gz
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@12.17.0 | linux | x64
gyp info ok
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@12.17.0 | linux | x64
gyp ERR! configure error
gyp ERR! stack Error: Can't find Python executable \"python\", you can set the PYTHON env variable.
gyp ERR! stack     at PythonFinder.failNoPython (/opt/xen-orchestra/node_modules/node-gyp/lib/configure.js:484:19)
gyp ERR! stack     at PythonFinder.<anonymous> (/opt/xen-orchestra/node_modules/node-gyp/lib/configure.js:406:16)
gyp ERR! stack     at F (/opt/xen-orchestra/node_modules/which/which.js:68:16)
gyp ERR! stack     at E (/opt/xen-orchestra/node_modules/which/which.js:80:29)
gyp ERR! stack     at /opt/xen-orchestra/node_modules/which/which.js:89:16
gyp ERR! stack     at /opt/xen-orchestra/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /opt/xen-orchestra/node_modules/isexe/mode.js:8:5
gyp ERR! stack     at FSReqCallback.oncomplete (fs.js:167:21)
gyp ERR! System Linux 5.4.0-33-generic
gyp ERR! command \"/usr/local/bin/node\" \"/opt/xen-orchestra/node_modules/node-gyp/bin/node-gyp.js\" \"configure\" \"--fallback-to-build\" \"--module=/opt/xen-orchestra/node_modules/argon2/lib/binding/argon2.node\" \"--module_name=argon2\" \"--module_path=/opt/xen-orchestra/node_modules/argon2/lib/binding\" \"--napi_version=6\" \"--node_abi_napi=napi\" \"--napi_build_version=0\" \"--node_napi_label=node-v72\"
gyp ERR! cwd /opt/xen-orchestra/node_modules/argon2
gyp ERR! node -v v12.17.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute '/usr/local/bin/node /opt/xen-orchestra/node_modules/node-gyp/bin/node-gyp.js configure --fallback-to-build --module=/opt/xen-orchestra/node_modules/argon2/lib/binding/argon2.node --module_name=argon2 --module_path=/opt/xen-orchestra/node_modules/argon2/lib/binding --napi_version=6 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v72' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/opt/xen-orchestra/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:315:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1021:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
node-pre-gyp ERR! System Linux 5.4.0-33-generic
node-pre-gyp ERR! command \"/usr/local/bin/node\" \"/opt/xen-orchestra/node_modules/argon2/node_modules/.bin/node-pre-gyp\" \"install\" \"--fallback-to-build\"
node-pre-gyp ERR! cwd /opt/xen-orchestra/node_modules/argon2
node-pre-gyp ERR! node -v v12.17.0
node-pre-gyp ERR! node-pre-gyp -v v0.13.0
node-pre-gyp ERR! not ok
$ scripts/run-script prepare
```

**Comment by Danp2** [comment]
Reverted script to use Python2-minimal for now

**Comment by Jarli01** [comment]
This looks like the source is missing, do any of those links resolve if you open them in a browser?

**Comment by Danp2** [comment]
I believe it tries to download pre-built binaries and then reverts to building from source when they can't be found. From what I can tell, the issue will resolve itself once a newer version of argon2 is required. See my discussion with @julien-f [here](https://xcp-ng.org/forum/topic/3107/python3-support).

**Comment by Danp2** [comment]
Recent updates to XO should now allow for python to be upgraded. Need to test to be sure.

**Comment by Jarli01** [comment]
Running a test build now with python3 to see what happens.

**Comment by Danp2** [comment]
Closed per #91
